### PR TITLE
removing not used methods

### PR DESCRIPTION
**Pull request by lucianosousa**
because less code is always better :)

### CODE REVIEW

File: app/models/rubygem.rb [hunk 1]
```diff
@@ -261,11 +257,6 @@ def find_or_initialize_version_from_spec(spec)
     version
   end
 
-  def monthly_downloads
-    key_dates = self.class.monthly_dates.map(&:to_s)
```
Comment (arthurnn) [inline app/models/rubygem.rb]: after removing this, do we still need `monthly_dates` method

Comment (farukaydin) [inline app/models/rubygem.rb]: don't need anymore, and also we should remove ```monthly_short_dates``` method with corresponding tests.

**Comment by arthurnn** [comment]
rubocop is failling for this

**Comment by lucianosousa** [comment]
done and rebase in one commit

**Comment by arthurnn** [comment]
thanks

**Comment by farukaydin** [comment]
:tada: :tada:

**Pull request merged**